### DISCUSSION TITLE
Disappointed and even very angry with Education's automatic review system.

### DISCUSSION THREAD

**Discussion by Apolsus** [community/community#58010] [GitHub Education]
### Hi everyone,

There are many Github in mainland China, but Chinese students cannot obtain normal student certification.

I tried to submit it 14 times. Whether it is a student ID card, student status certificate, or China's official online verification report, I can't prove that I am a student. Despite my name, the graduation date is largely written there. Also provides verification codes to assist in verification.

But Github doesn't think you're a student. Why can I get a student certification from Apple but not a student certification on Github?

So, f**k Q Github!

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
